### ISSUE TITLE
Upgrading to 0.4.4 SQL error using InnoDB

### ISSUE THREAD

**Issue by marxxx** [owncloud/mail#1492]
### Steps to reproduce
1. Upgrade OC Mail app to 0.4.4 using MySQL InnoDB as database.

### Expected behaviour
App should update.

### Actual behaviour
Getting SQL error: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes
InnoDB max key length is 255. At the moment app tries to increase it to 256.

**Comment by pvanek** [comment]
the full occ upgrade output:

```
Update 3rd-party app: mail
Doctrine\DBAL\Exception\DriverException: An exception occurred while executing 'ALTER TABLE oc_mail_collected_addresses CHANGE `email` `email` VARCHAR(256) NOT NULL':
                                                                                                                                                                              
SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes                                                               
Update failed
```

**Comment by ChristophWurst** [comment]
Hi,
this is a duplicate of #1480. This was fixed in #1484, which will be released with 0.4.5
